Improve Tardigrade launch blog (#289)
trinodb/trino.io: _posts/2022-05-05-tardigrade-launch.md

If running fast batch jobs on the fastest state-of-the-art query engine 
interests you, consider playing around with the tutorial above and giving us 
your feedback. You can reach us on the [#project-tardigrade](https://bit.ly/3IFlNXy) 
channel in our [Slack](https://trino.io/slack.html). 

> Maybe you even want to write about your experience and results, or become a 

Reviewer: ```suggestion
If you would like to write about your experience and results, or become a 
```

> Maybe you even want to write about your experience and results, or become a 
> contributor. Let us know on the [#project-tardigrade](https://bit.ly/3IFlNXy) 
> channel in the Trino Slack. We are happy to send you Tardigrade swag as a 
> thank you.

Reviewer: ```suggestion
IF you would like to write about your experience and results, or become a 
contributor, also let us know on the [#project-tardigrade](https://bit.ly/3IFlNXy). We are happy to send you Tardigrade swag as a  thank you.
```
